test/e2e/kubernetes/pod/pod.go

```diff
@@ -120,13 +120,21 @@ type Status struct {
 }
 
 // CreatePodFromFile will create a Pod from file with a name
-func CreatePodFromFile(filename, name, namespace string) (*Pod, error) {
-	cmd := exec.Command("kubectl", "apply", "-f", filename)
-	util.PrintCommand(cmd)
-	out, err := cmd.CombinedOutput()
-	if err != nil {
-		log.Printf("Error trying to create Pod %s:%s\n", name, string(out))
-		return nil, err
+func CreatePodFromFile(filename, name, namespace string, retries int) (*Pod, error) {
```

Comment:
Would it make sense to run the retries concurrently and return as soon as a successful pod is registered in the channel?